src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -53,7 +53,7 @@ public void execute_personAcceptedByModel_addSuccessful() throws Exception {
     }
 
     /**
-     * tests for duplicate persons
+     * Tests for duplicate persons.
      */
     @Test
```

Comment:
Slightly confused why we are updating the previous codebase. Is it failing checkstyle?